Views/backend/connect/view/config/import/form.js

```diff
@@ -219,12 +225,34 @@ Ext.define('Shopware.apps.Connect.view.config.import.Form', {
                             xtype: 'checkbox',
                             name: 'importImagesOnFirstImport',
                             fieldLabel: me.snippets.importPicturesLabel,
-                            helpText: me.snippets.importPicturesHelp,
                             inputValue: 1,
                             uncheckedValue: 0,
                             labelWidth: me.defaults.labelWidth,
                             listeners:{
                                 change: function(checkbox, newValue, oldValue, opts){
+                                    if (checkbox.getValue() === false) {
+                                        Ext.Ajax.request({
+                                            url: '{url controller=ConnectConfig action=checkCronPlugin}',
+                                            method: 'GET',
+                                            success: function (response, opts) {
+                                                var data = Ext.JSON.decode(response.responseText);
+                                                if (data.cronActivated !== true) {
+                                                    checkbox.setValue(true);
```

Comment:
so if the cron plugin is not active I'm not able to uncheck option "Import images during initial import?"

Reply:
exactly because if deactivating this option without having CronPlugin installed it would be like saying "I don't want to import images" and that is probably not what the user wants
if someone wants that i think we need to make an explicit option for that